hail/python/hailtop/hailctl/auth/create_user.py

```diff
@@ -20,25 +20,22 @@ def init_parser(parser):
 
 
 async def async_main(args):
```

Comment:
why make these changes? Do we report the username in the underlying exception?

Reply:
The linter complained about using too generic an error class, it wants us to subclass `Exception`. I removed the try except because I felt like it wasn't actually any more helpful than whatever stack trace was underneath, but I didn't think about the username not being in there. Can add that back in if you want.